bundler.plugins.d/foreman_ansible_core.rb

```diff
@@ -0,0 +1 @@
+gem 'foreman_ansible_core'
```

Comment:
Sorry, that confused me a bit. How do you enforce that `foreman_ansible_core` is loaded with this plugin (or `smart_proxy_dynflow`)? Currently from what I see it's up to the user to set this line in bundler.plugins.d in the proxy.